sdk/metricsadvisor/ai-metrics-advisor/samples-dev/incidentsAndAlerts.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 /**
- * This sample demonstrates how to query incidents.
+ * @summary This sample demonstrates how to query incidents.
  */
```

Comment:
I would just take a pass over the source files to make sure that the block comments have spacing around them. It looks cleaner.

```suggestion
 */
 
```

Reply:
done